app/models/user.rb

```diff
@@ -59,8 +59,7 @@ class User < ApplicationRecord
 
   belongs_to :role, optional: true
   has_many :ratings, dependent: :destroy
-
```

Comment:
Why you deleted empty line?

Reply:
it's prettier

Reply:
I don t ask you revert it. But please try create more accurate commits!